Packs/Code42/Integrations/Code42/Code42.yml

```diff
@@ -419,11 +419,20 @@ script:
       description: The username of the High Risk Employee.
       type: Unknown
   - arguments:
-    - default: false
+    - auto: PREDEFINED
+      default: false
       description: To filter results by employees who have these risk tags. Space
```

Comment:
The description of all these fields still say they are space delimited.

Reply:
fixed